### PR TITLE
Added interval, Updated Plane, Added Enzyme

### PR DESCRIPTION
**Pull request by justinamaple**

### CODE REVIEW

File: src/containers/Desk.js [hunk 1]
```diff
@@ -7,47 +7,72 @@ import Write from '../components/Write'
 class Desk extends Component {
   state = {
     letterStack: [],
-    logged_in: null
+    loggedIn: null,
```
Comment (kara-o) [inline src/containers/Desk.js]: do we need loggedIn in state here when we are already passing down accountId from App (and therefore already logged in)?

File: src/containers/Desk.js [hunk 2]
```diff
@@ -7,47 +7,72 @@ import Write from '../components/Write'
...
   renderWrite = () => {
-    console.log('write')
     return <Write />
   }
 
-  renderRead = e => {
-    console.log(e.target.id, this)
-    return <Read />
+  renderRead = letter => {
+    // TODO: mark the letter as seen for the given account
```
Comment (kara-o) [inline src/containers/Desk.js]: awesome, thanks, about to work on this component

File: src/containers/Desk.js [hunk 3]
```diff
@@ -7,47 +7,72 @@ import Write from '../components/Write'
...
+    return <Read letter={letter} />
   }
 
-  renderPlanes = () => {
-    return this.state.letterStack.map(letter => {
-      return <Plane key={letter.id} letter={letter} handleClick={this.renderRead} />
-    })
+  renderPlane = () => {
+    if (this.state.plane)
+      return <Plane key={this.state.plane.id} plane={this.state.plane} handleClick={this.renderRead} />
```
Comment (kara-o) [inline src/containers/Desk.js]: for handleClick - i made a handlePlaneClick function that will setState isRead: true, and then call renderRead with the plane/letter object

File: src/containers/Desk.js (hunk 1 above)
Comment (justinamaple) [reply src/containers/Desk.js]: You are totally right, we don't need to save it here, good catch!

File: src/containers/Desk.js (hunk 3 above)
Comment (justinamaple) [reply src/containers/Desk.js]: Perfect! Sounds like it should be an easy merge

**Comment by justinamaple** [comment]
Closing since Kara had a big push, merged with hers and put at new PR: https://github.com/justinamaple/KindWordsFrontEnd/pull/5

**Pull request closed without merging**